From 5509409f48d969ee9e300fadecf545c095a10f13 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Mon, 16 May 2022 15:11:50 +0300
Subject: [PATCH 18/30] net: s32cc-gmac: Sync bindings

Use the same bindings for S32CC GMAC as for Linux.

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/mach-s32/s32-cc/eth.c  | 24 ++++++++++++------------
 drivers/net/dwc_eth_qos_s32cc.c |  2 +-
 2 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/arch/arm/mach-s32/s32-cc/eth.c b/arch/arm/mach-s32/s32-cc/eth.c
index 828f4bd918..3270a773e0 100644
--- a/arch/arm/mach-s32/s32-cc/eth.c
+++ b/arch/arm/mach-s32/s32-cc/eth.c
@@ -3,28 +3,28 @@
  * Copyright 2019-2022 NXP
  */
 #include <common.h>
+#include <clk.h>
 #include <dm.h>
-#include <asm/io.h>
+#include <fdt_support.h>
+#include <hwconfig.h>
+#include <malloc.h>
 #include <net.h>
+#include <phy.h>
+#include <s32gen1_clk_utils.h>
+#include <s32gen1_gmac_utils.h>
+#include <asm/io.h>
+#include <asm/types.h>
+#include <dm/device_compat.h>
+#include <dm/pinctrl.h>
 #ifndef CONFIG_DM_ETH
 #include <netdev.h>
 #endif
-#include <phy.h>
-#include <malloc.h>
-#include <asm/types.h>
 #if CONFIG_IS_ENABLED(DWC_ETH_QOS_S32CC)
 #include <dm/platform_data/dwc_eth_qos_dm.h>
 #endif
 #if CONFIG_IS_ENABLED(FSL_PFENG)
 #include <dm/platform_data/pfeng_dm_eth.h>
 #endif
-#include <fdt_support.h>
-#include <clk.h>
-#include <s32gen1_clk_utils.h>
-#include <s32gen1_gmac_utils.h>
-#include <dm/device_compat.h>
-#include <dm/pinctrl.h>
-#include <hwconfig.h>
 
 #if CONFIG_IS_ENABLED(FSL_PFENG)
 static void ft_update_eth_addr_by_name(const char *name, const u8 idx,
@@ -299,7 +299,7 @@ static int get_gmac_clocks(u32 mode, const char **rx,
 			   const char **tx, const char **ts)
 {
 	if (ts)
-		*ts = "ts";
+		*ts = "ptp_ref";
 
 	switch (mode) {
 	case PHY_INTERFACE_MODE_SGMII:
diff --git a/drivers/net/dwc_eth_qos_s32cc.c b/drivers/net/dwc_eth_qos_s32cc.c
index f3e7c6e742..d042e6e394 100644
--- a/drivers/net/dwc_eth_qos_s32cc.c
+++ b/drivers/net/dwc_eth_qos_s32cc.c
@@ -310,7 +310,7 @@ static int eqos_stop_resets_s32cc(struct udevice *dev)
 
 static ulong eqos_get_tick_clk_rate_s32cc(struct udevice *dev)
 {
-	return s32gen1_get_dev_clk_rate("axi", dev);
+	return s32gen1_get_dev_clk_rate("stmmaceth", dev);
 }
 
 static int eqos_calibrate_pads_s32cc(struct udevice *dev)
-- 
2.17.1

